src/System.Runtime/src/project.json

```diff
@@ -1,12 +1,15 @@
 {
   "frameworks": {
+    "netstandard1.7": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.CoreCLR": "1.0.2-beta-24325-03"
+      }
+    },
     "netstandard1.5": {
+      "__comment": "This should be a stable version of the TargetingPack package that matches netcoreapp1.0 RTM",
```

Comment:
I believe we didn't actually ship a stable version of the Targeting package, so the work would be to select one that matches netcoreapp and re-publish it with a stable version on it, and then add it here. And no, there is no reason why we can't make that now, just wanted to get this PR out first to get feedback.